### ISSUE TITLE
Transaction stuck, unconfirmed bitcoins.

### ISSUE THREAD

**Issue by stackbabbler** [Multibit-Legacy/multibit-hd#969]
I would greatly appreciate any help on solving this problem. The btc is stuck and it seems that the problem gets larger whatever I try to do to fix it.

1. Transferred 0.4 BTC from localbitcoins to MultiBit HD 0.4.1
2. Sent 0.1 btc to poloniex. I didn´t know about fees and apparently my fee was too low so the btc got stuck in limbo. I also sent some small amounts to both poloniex and a different computer to see if it worked.
3. Let the wallet try and send the 0.1 btc for four days, still stuck.
4. Repaired the wallet. Suddenly all my btc is unconfirmed and only 0.0039 btc is available to use. 
5. Tried backing up the roaming folder, and backup folder. Then deleting the folders and recovering from wallet words. If I backed up to the oldest date, then it looked like everything was okay, until I tried to send any btc and I got an error that I have insufficient btc and everything turns into unconfirmed again. I tried this over ten times.
6. Completely removed MultiBit HD 0.4.1, installed version 0.1.1 instead. Tried first wallet words, still unconfirmed. Tried repairing, still unconfirmed. Tried using the backup file, still unconfirmed.
7. Tried Multibit HD 0.1.0 also. Here I have 0.0039 btc available, 0.299 btc unconfirmed and 0.1 btc sending. Blockchain alerts me that my transactions are attempts at double spending. I´ve had this wallet open for 24 hours, the original transaction from localbitcoins is confirmed by 877 blocks, yet the wallet claims it´s unconfirmed. The stuck low-fee sending transaction to poloniex has been seen by 13 peers.

This is the first transaction that I got from localbitcoins:
https://blockchain.info/tx-index/bc5fb3eeaaeddaa781d61b900052c18a36b2195ad1773d9f8c1ad36c136939c1

This is the stuck transaction:
https://blockchain.info/tx-index/f0eedd44974b8bf678a403d45f1aa56e31b9bb3300c17976d73320e62dc8e65f

I have no idea what I can do next as I´ve tried everything I´ve come across. 

Thank you.

**Comment by stackbabbler** [comment]
After days of frustration I finally managed to get my coins out of Multibit HD. I installed 0.4.1 again and randomly went back and forth trying different backups. I had tried so many different things that I forgot to mention that I also tried to "Empty wallet" into a Multibit Classic on my other computer, but it only gave me the insufficient funds error. This one backup I found remembered that transfer and immediately began sending the btc, and now I have my btc back on my other computer.

Devs should really fix this problem, I see that others have had similar experiences and my solution only worked with one out of tens of backup files. I'm still using Multibit Classic without any problems.

**Comment by Kcchouette** [comment]
I switch to version 0.5.1 and now it's fixed for me

**Comment by alstef** [comment]
Related issues
https://github.com/keepkey/multibit-hd/issues/862
https://github.com/keepkey/multibit-hd/issues/947
https://github.com/keepkey/multibit-hd/issues/948
https://github.com/keepkey/multibit-hd/issues/946

@kliyer please stop closing these very serious issues without an actual fix, people are loosing money and this has nothing to do with KeepKey Support. It's a connectivity problem of some sort because MBHD can't correctly identify Blockchain confirmed transactions or something like that.

Bottom line, serious issue, confirmed, needs attention. Thanks!
